docs/docs/getting-started/installing-nebari.md

```diff
@@ -64,9 +64,26 @@ If you are unable to successfully validate the Nebari installation above, you ma
 :::
 
 ---
-
 ## What's next?
 
+### Nebari Init and Guided Init
+
+Nebari Init create and initialize your Nebari configuration. Guided init does the same but user can have different experience while running it (more enhanced).
+
+You can pass the `--help` flag to the `init` command to check all the arguments and options available for it.
+
+```bash
+nebari init --help 
+```
+
+![A representation of the output generated when Nebari init help command is executed, the output contains a list of the available options and arguments and their use.](/img/getting-started/nebari-init-help-2.png "Nebari's init help command line output")
+
+:::tip
+You can pass the `--help` flag with the `--guided-init` flag to the `init` command to check all the arguments and options available for it.
```

Comment:
Do we mean `nebari init --help --guided-init` here? This command starts the guided init wizard for me right, I'm not sure if that's intentional or a bug.

If it's expected, I think we may need to update this tip message.

Reply:
Yes, you are right. I have added the reference to the guided-init command in the tip section.

Reply:
Thanks!